.github/workflows/continuous-integration.yml

```diff
@@ -178,13 +178,7 @@ jobs:
           - "7.4"
         postgres-version:
```

Comment:
I noticed that we have a class named `lib/Doctrine/DBAL/Platforms/PostgreSQL91Platform.php` in our codebase, should it be removed?